src/libraries/System.Text.RegularExpressions/tests/System.Text.RegularExpressions.Tests.csproj

```diff
@@ -28,20 +28,17 @@
     <Compile Include="RegexCompilationHelper.cs" />
     <Compile Include="RegexCultureTests.cs" />
     <Compile Include="RegexMatchTimeoutExceptionTests.cs" />
-    <Compile Include="RegexParserTests.cs" />
     <Compile Include="RegexReductionTests.cs" />
-    <Compile Include="..\src\System\Text\RegularExpressions\RegexParseError.cs"
-             Link="System\Text\RegularExpressions\RegexParseError.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">
+    <Compile Include="RegexParserTests.cs" />
     <Compile Include="GroupCollectionReadOnlyDictionaryTests.cs" />
     <Compile Include="CaptureCollectionTests2.cs" />
     <Compile Include="GroupCollectionTests2.cs" />
     <Compile Include="MatchCollectionTests2.cs" />
     <Compile Include="PrecompiledRegexScenarioTest.cs" />
     <Compile Include="RegexCompilationInfoTests.cs" />
     <Compile Include="RegexGroupNameTests.cs" />
-    <Compile Include="$(CommonTestPath)System\Diagnostics\DebuggerAttributes.cs"
-             Link="Common\System\Diagnostics\DebuggerAttributes.cs" />
+    <Compile Include="$(CommonTestPath)System\Diagnostics\DebuggerAttributes.cs" Link="Common\System\Diagnostics\DebuggerAttributes.cs" />
```

Comment:
(nit) I've never known why, but in all my PRs, folks insist that I don't change the existing formatting here :)

Reply:
Oops, must've missed it on the compare-before-commit review locally. Fixed!